style.css

```diff
@@ -0,0 +1,66 @@
+@import url('https://fonts.googleapis.com/css2?family=Inter:wght@400;600;700&display=swap');
+
+* {
+  box-sizing: border-box;
+  margin: 0;
+  font-family: Inter, Times, serif;
+}
+
+.main-section-container {
+  background-image: url("/assets/images/bg1.png");
+  background-repeat: no-repeat;
+  background-size: cover;
+  display: flex;
+  flex-direction: column;
+  padding: 2.4rem;
+  height: 100vh;
+}
+
+.header-container {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  width: 100%;
+  padding: 2rem;
+  font-weight: 600;
+}
+
+.welcome {
+  color: black;
+  font-weight: 600;
+  font-size: 20px;
+}
+
+.menu-btn:hover {
+  transform: scale(1.5);
+}
+
+.main-title {
```

Comment:
Please ensure that you stick to the design as much as possible (e.g. font, colors, images, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1).

Your Design | Figma Design
:---:|:---:
 
![your_design](https://user-images.githubusercontent.com/10332499/153521475-863f29c8-ab4a-41b8-9a12-d6538eb8c88c.PNG) | ![figma_design](https://user-images.githubusercontent.com/10332499/153521509-da6e8c0e-a578-4c44-87e0-c3661a5491c0.PNG)